### ISSUE TITLE
Uncaught TypeError: url.URL is not a constructor

### ISSUE THREAD

**Issue by aquiladev** [ConsenSysDiligence/armlet#103]
I have an error on application startup when I import armlet package
![image](https://user-images.githubusercontent.com/1164492/56462927-89a03680-63cb-11e9-8428-530ce2f36ea8.png)

$ npm -v
6.4.1
$ node -v
v10.15.1

**Comment by muellerberndt** [comment]
At first sight simply replacing url.URL with URL and not explicitly importing the url package seems to work fine. Is there any reason this could break stuff elsewhere?

**Comment by aquiladev** [comment]
does not work in my case
removing the import of url `const url = require('url')` works (node v10.15.1)


/util.js
```
/**
 * Wait for the specified time.
 *
 * @param {Number} time Interval duration [ms].
 * @return {Promise} Resolves after the specified delay.
 */
exports.timer = async time =>
  new Promise(resolve => setTimeout(resolve, time))

exports.joinUrl = (base, path) => {
  const u = new URL(path, base)
  return u.href
}
```

**Comment by muellerberndt** [comment]
I put in another PR which I think fixes the issue, see #105 #106.

**Comment by vdenisov-pro** [comment]
The problem occurs because `url.URL` is `undefined` (read more in this [comment](https://github.com/ConsenSys/armlet/issues/105#issuecomment-673060135)).
